flask/auth.py

```diff
@@ -43,11 +43,14 @@ def generate_uuid():
 @swag_from("documentation/login.yaml", methods=["POST"])
 def login():
     salt = env_var["salt"]
-    email = request.form.get("email")
+    username = request.form.get("email")
```

Comment:
The form-data key that is "email" here is indicating that we only require email and not username in the code.

So, can you name it something more generic on both flask and react side.

For example this.

```
user_identifier = require.form.get("user_identifier")
```

And then,

```
query = {"$and": [{"$or": [{"email": user_identifier}, {"username":user_identifier}]}, {"password": hashed_password}]}
```

Reply:
sure.